content/en/tracing/trace_collection/library_config/_index.md

```diff
@@ -81,6 +81,64 @@ The following configuration options behave consistently across the latest versio
 **Supported Input**: A string representing an application environment name (for example, `prod`, `dev`) <br>
 **Description**: Adds an environment tag to all spans generated by the tracer instance.
 
+`DD_TAGS`
+: **Default**: `null` <br>
+**Supported Input**: A string representing key value pairs that are delimited by a colon and separated by a comma and/or a space (for example, `<key1>:<value1>, <key2>:<value2>`>) <br>
+**Description**: Tags to apply to produced data. Must be a list of `<key>:<value>` separated by commas and/or spaces.
+
+`DD_TRACE_SAMPLING_RULES`
+: **Default**: `null` <br>
+**Supported Input**: A JSON string representing an array of sampling rules. Each rule must contain a `sample_rate` (float between 0 and 1) and can optionally include a `service` (string) and `name` (string) pattern to match. Example: `[{"sample_rate": 0.1, "service": "my-service", "name": "http.request"}]` <br>
```

Comment:
Our existing config pages have text that says `For more information, see [Ingestion Mechanisms][2]` with a link to `/tracing/trace_pipeline/ingestion_mechanisms/`. I think it would be helpful to include that here as well.